src/universal/modules/teamDashboard/components/ArchiveTeamConfirmation/ArchiveTeamConfirmation.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component, PropTypes} from 'react';
+import {reduxForm, Field} from 'redux-form';
+import InputField from 'universal/components/InputField/InputField';
+import {cashay} from 'cashay';
+import Button from 'universal/components/Button/Button';
+import {css} from 'aphrodite-local-styles/no-important';
+import appTheme from 'universal/styles/theme/appTheme';
+import withStyles from 'universal/styles/withStyles';
+
+const styleThunk = () => ({
+  errorMessage: {
+    color: appTheme.palette.warm
+  }
+});
+
+@withStyles(styleThunk)
+@reduxForm({form: 'archiveTeamConfirmation'})
+export default class ArchiveTeamConfirmation extends Component {
+
+  static propTypes = {
+    teamName: PropTypes.string.isRequired,
+    teamId: PropTypes.string.isRequired,
+    handleSubmit: PropTypes.func.isRequired,
+    router: PropTypes.object.isRequired,
+    styles: PropTypes.object.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      showConfirmationField: false,
+      showError: false
+    };
+  }
+
+  archiveTeam = () => {
+    const {teamName, teamId, router} = this.props;
+    const variables = {
+      updatedTeam: {
+        id: teamId,
+        name: teamName,
+        isArchived: true
+      }
+    };
+    cashay.mutate('archiveTeam', {variables});
+    router.push('/me');
+  }
+
+  formSubmit = (data) => {
+    const {teamName} = this.props;
+    const {archivedTeamName} = data;
+    if (teamName === archivedTeamName) {
+      this.archiveTeam();
+    } else {
+      this.setState({showError: true});
+    }
+  }
+
+  formBlurred = () => {
+    this.setState({
+      showConfirmationField: false,
+      showError: false
+    });
+  }
+
+  archiveTeamClick = () => {
+    this.setState({showConfirmationField: true});
+  }
+
+  render() {
+    const {handleSubmit, styles} = this.props;
+    const {showConfirmationField, showError} = this.state;
+    return (
+      <div>
+        {showError &&
+          <div className={css(styles.errorMessage)}>
+            The team name entered was incorrect
+          </div>
+        }
+        {!showConfirmationField ?
```

Comment:
i think using a modal here might look here. Once they click the button, we want to draw all their attention to a single task at hand, which fits the bill for when to use a modal. thoughts?